pkgs/by-name/ne/nexusmods-app/package.nix

```
@@ -127,5 +130,29 @@ buildDotnetModule rec {
       MattSturgeon
     ];
     platforms = lib.platforms.linux;
+    description = "Game mod installer, creator and manager";
+    longDescription = ''
+      ${meta.description}.
+
```

```suggestion
```
longDescription should not just repeat description as they are often rendered next to each other

---

Are they _always_ rendered together? Would including the more verbose description from upstream be ok?

---

[Upstream's readme](https://github.com/Nexus-Mods/NexusMods.App/blob/main/README.md) is as follows:

> Nexus Mods app is a mod installer, creator and manager for all your popular games.
> 
> Easy to use, runs on your standard Windows PC and Linux alike. Don't waste time troubleshooting, play your games,
> fill those knees with arrows and most importantly, ***Have Fun***!
> 
> Learn more about the App on the [Wiki](https://nexus-mods.github.io/NexusMods.App/)

Perhaps the long description could include some of that along with a note that the app is still experimental?

e.g.:

```suggestion
      A mod installer, creator and manager for all your popular games.

      Currently experimental and undergoing active development,
      new releases may include breaking changes!

```